File: src/main/java/me/liuli/falcon/check/checks/killaura/KAHit1.java
package me.liuli.falcon.check.checks.killaura;

import cn.nukkit.Player;
import cn.nukkit.network.protocol.DataPacket;
import me.liuli.falcon.check.CheckBase;
import me.liuli.falcon.check.CheckType;

public class KAHit1 extends CheckBase {
    public KAHit1() {
        super(CheckType.KillAura,"hit","HIT1");
    }

    @Override
    public void onReceivePacket(Player player,DataPacket packet){

    }
}

File: src/main/java/me/liuli/falcon/check/checks/killaura/KAHit1.kt
package me.liuli.falcon.check.checks.killaura

import cn.nukkit.network.protocol.DataPacket
import me.liuli.falcon.FalconAC
import me.liuli.falcon.check.CheckBase
import me.liuli.falcon.check.CheckType
import me.liuli.falcon.data.PlayerData

class KAHit1 : CheckBase(CheckType.KillAura, "hit", "HIT1") {
    override fun onSendPacket(data: PlayerData, packet: DataPacket) {
        FalconAC.INSTANCE.logger.info("SEND_PLAYER="+data.player.name+", PACKET="+packet)
    }

    override fun onReceivePacket(data: PlayerData, packet: DataPacket) {
        FalconAC.INSTANCE.logger.info("RECEIVE_PLAYER="+data.player.name+", PACKET="+packet)
    }
}